### ISSUE TITLE
Certificate update Portal/Server - with InstallLicenseConfigure - Noe respons in reading new certificate  

### ISSUE THREAD

**Issue by grumor** [Esri/arcgis-powershell-dsc#561]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Module Version

* 4.3.0

### Affected Resource(s)

* ArcGIS_Server
* ArcGIS_Portal

### Configuration Files

```
{
	"NodeName": "servermachine",
	"Role": [
		"Server",
		"Portal"
	],
	"SslCertificates": [
		{
			"Path": "\\\\fileserver\\share$\\Certificates\\cert_file.pfx",
			"Password": "REDACTED",
			"CNameFQDN": "servermachine",
			"Target": [
				"Server",
				"Portal"
			],
			"SslRootOrIntermediate": [
				{
					"Alias": "ROOT_ALIAS",
					"Path": "\\\\fileserver\\share$\\Certificates\\ca_root_cert_file.cer"
				}
			]
		}
	]
}
```

### Expected Behavior
New renewed certificate with new Thumbprint, read from fileshare and exchanged with existing certificate on Portal/Server

### Actual Behavior
Nothing happens - No log entries, no steps in - debugSwitch written. It seams like the certificate is not read at all.  

### Steps to Reproduce
Renew Certificate from CA Authority - Replace the certificate on fileshare (Rename old certificate _old. Give new certificate the same name as the origin certificate

run - -Mode InstallLicenseConfigure

**Comment by cameronkroeker** [comment]
Hi @grumor, 

Are you also passing in the **ConfigData.Portal** and **ConfigData.Server** attributes in the json configuration file?

Thanks,
Cameron K.

**Comment by grumor** [comment]
Yes. I do. I think you can close this. I have tested this in another environment and it worked as expected. I think the reason st where i got the problem is the certificate it selves. The certificate is valid, but its not readable (you are only able to delete it by adding /delete in the admin). I know there is some use of characters that Esri dont like. I need to figure out that first. Before persue this issue.

Sendt fra Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
